docs/gh-actions/gh-update.md

```diff
@@ -78,7 +84,9 @@ The section continues immediately after the [Update Fork](#update-fork) section.
 
 #### What if the Build Fails
 
-There are some updates where you need to run some actions before build. If a new release is announced, look to see if there are instructions about this. In those cases, you will probably be told to run Action 2: Add Identifiers and Action 3: Create Certificates before Action 4: Build Loop. Those steps are fast and there is no harm in trying them.
+If a new release is announced, look to see if there are instructions about extra steps required with the release. (The release after 3.2.2 will certainly have extra instructions.)
```

Comment:
Perhaps a link to the [release page](https://loopkit.github.io/loopdocs/version/releases/) would be of help.